README.md

```diff
@@ -9,7 +9,7 @@ with either `9.2`, `10.0`, or `10.1`. Also replace `<processor type>`
 with `cpu` or `gpu`.
 
 ```bash
-conda create -n ucx -c conda-forge -c conda-forge/label/rc_ucx cudatoolkit=<CUDA version> ucx-proc=*=<processor type> ucx ucx-py python=3.7
+conda create -n ucx -c conda-forge -c rapidsai cudatoolkit=<CUDA version> ucx-proc=*=<processor type> ucx ucx-py python=3.7
```

Comment:
Should this be `rapidsai-nightly` for awhile?

We're calling the release alpha, so perhaps we should even include a comment saying that you likely want bugfixes/improvements from the nightly packages until we have a more stable release.